blog/_posts/2019-01-18-fluxdiffeq.md

```diff
@@ -0,0 +1,650 @@
+---
+layout: post
+title:  FluxDiffEq.jl: A Julia Library for Neural Differential Equations
+author: Chris Rackauckas, Mike Innes, Yingbo Ma, Lyndon White, Vaibhav Dixit
+---
+
+In this blog post we
+will show you how to easily, efficiently, and
+robustly add differential equation solvers to
+neural networks in Julia.
+
+![Flux ODE Training Animation](https://user-images.githubusercontent.com/1814174/51399500-1f4dd080-1b14-11e9-8c9d-144f93b6eac2.gif)
+
+The [Neural Ordinary Differential Equations](https://arxiv.org/abs/1806.07366)
+paper has been a hot topic even before it made a splash as Best Paper of
+NeuralIPS 2018. By mixing ordinary differential equations and neural networks
+they were able to improve training speeds and accuracy over ResNet. Now with the

```

Comment:
We don't make any claims of improved training speeds or accuracy over a comparably sized ResNet. The win is fewer parameters and O(1) memory to achieve similar performance as ResNet:
![image](https://user-images.githubusercontent.com/4240549/51410744-8169fe00-1b33-11e9-88cc-160eff25a9b4.png)